examples/morpheusvm/tests/e2e/e2e_test.go

```diff
@@ -47,6 +47,9 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 
 	// Run only once in the first ginkgo process
 	nodes := tmpnet.NewNodesOrPanic(flagVars.NodeCount())
+
```

Comment:
Could we replace lines 50-68 with updating this function https://github.com/ava-labs/hypersdk/blob/d6377a4535c9f4c62ec4eea48665999641ef3b7d/tests/fixture/environment.go#L13 ?

Then each VM can rely on a fixed URL without replicating this bit of code in their `SynchronizedBeforeSuite` code.